pkg/scheduler/cache/cluster_info/cluster_info.go

```diff
@@ -414,7 +420,7 @@ func (c *ClusterInfo) snapshotTopologies() ([]*kueue.Topology, error) {
 }
 
 func getDefaultPriority(dataLister data_lister.DataLister) (int32, error) {
```

Comment:
I have deja vu from https://github.com/NVIDIA/KAI-Scheduler/pull/526/files#diff-a4d20671885379d4aaca001082a6e14c91251db3bdc4e4c41de7da0a0a2d74ecR73 - please consolidate these two

Reply:
Yes, it is very much alike.
I thought about unifying these two but this one uses dataLister and the other place use a kubeClient, and half of this function logic is to list the priority classes using it.
Do we have a common interface for both?